edited the README, add index.html

AnatolyKalyuzhny opened the pull request.

AnatolyKalyuzhny committed:
edited the README, add index.html

AnatolyKalyuzhny committed:
added classes. added style.css

Yan1993-Dev commented on css/style.css:
```diff
@@ -0,0 +1,5 @@
+#rectangle {
```
По id не задаем стилизацию

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Document</title>
```
Тайтл должен быть заполнен

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+<body>
+    <header class="header"> 
+        <img src="" alt="" class="header__ logo">
+        <ul class="header__list">
+            <li class="header__list-item"><a href="#" class="header__list-link_">Blog</a></li>
+            <li class="header__list-item"><a href="#" class="header__list-link">Catalog</a></li>
+            <li class="header__list-item"><a href="#" class="header__list-link">About Us</a></li>
```
первый список не закрыт тегом

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+    </header>
+    <main>
+        <section class="banner">
+            <h1>01</h1>
```
01 это не заголовок 1-го уровня

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+    <main>
+        <section class="banner">
+            <h1>01</h1>
+            <div>
```
див тут не нужен

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+            <h1>01</h1>
+            <div>
+                <ul>
+                    <p><li id="rectangle"></li></p>
+                    <p><li id="rectangle"></li></p>
+                    <p><li id="rectangle"></li></p>
+                    <p><li id="rectangle"></li></p>
```
немного по другому, сначала идет li, а потом уже всё остальное. И тут не будет текста, тег p для текста. Сделай лучше кнопками

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+            <h2>Category Products</h2>
+            <ul>
+                <li><img src="" alt="">
+                <h3>APPAREL
```
Это не заголовки, а ссылки в категории, просто 3-х и 4-х нету

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+                <h3>ACCESSORIES</h3>
+                </li>
+            </ul>
+            <li>
+                <img src="" alt="">
+                <img src="" alt="">
+                <p>Modern finds for the modern home. Curated goods by unique designers around the world from fashion to furniture for kids and adults.</p>
+            </li>
```
li не может быть вне списка и список тут не нужен, можно сделать div и в него положить все элементы

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        <footer>
+            <section>
+                <h4>Navigation</h4>
+                <hr width="200px"
+                    align="left">
```
Полосу тегом не делаем, все, что видно, внешние какие либо элементы через css

AnatolyKalyuzhny committed:
made corrections by class

AnatolyKalyuzhny committed:
added styles.css/common.css

AnatolyKalyuzhny committed:
edited .css

AnatolyKalyuzhny committed:
edited .common

AnatolyKalyuzhny committed:
fixed:

for description container
-float
+flexbox
correct section designation (main and footer)

AnatolyKalyuzhny committed:
some changes
